src/ir_Denon.cpp

```diff
@@ -1,5 +1,12 @@
 // Copyright 2016 Massimiliano Pinto
 // Copyright 2017 David Conran
+/// @file
+/// @brief Denon support
+/// @see https://github.com/z3t0/Arduino-IRremote/blob/master/ir_Denon.cpp
+/// @see http://assets.denon.com/documentmaster/us/denon%20master%20ir%20hex.xls
+
+// Supports:
+//   Brand: Denon, Model: AVR-3801 ?
```

Comment:
`//   Brand: Denon, Model: AVR-3801 A/VReceiver`

https://www.google.com/search?q=AVR-3801

Reply:
Was not intended to be in this PR, Will have that in a separate one after going thru some of the undocumented files

Reply:
Also the ? here is because we are not sure if it actually does support that one, as it is just taken from one of the notes on what is marked as source for the code.

Reply:
ahhh